protostar/argument_parser/argument_parser_facade.py

```diff
@@ -210,3 +255,12 @@ def __init__(self, argument_name: str, command_name: Optional[str]) -> None:
             else f"Missing required argument: `{argument_name}`"
         )
         super().__init__(self.message)
+
+
+class InconsistentInputTypesException(Exception):
+    def __init__(self, previous_type: type, current_type: type) -> None:
+        self.message = (
+            f"Inconsistent input types: "
+            f"{str(previous_type)} and {str(current_type)} are not compatible"
```

Comment:
Why are they not compatible? I think it would be better to rewrite this error message as "Cannot mix `x=...` and `123` syntaxes". Or like Python's: 

![image](https://user-images.githubusercontent.com/3450050/201933104-6c2792b2-124a-40bd-a7f3-c027f93dd438.png)

Reply:
The former's ok, the latter would require detecting which one went first.

Reply:
[address feedback in argument parser facade](https://github.com/software-mansion/protostar/pull/1140/commits/2dcc09754a7c9b31bb5ccb5ceedf8e8e6e625ffd)